docs/application_packages.md

```diff
@@ -28,7 +28,7 @@ appropriate values (make sure this is all on one line):
 
     $ mvn archetype:generate \
      -DarchetypeGroupId=org.apache.apex \
-     -DarchetypeArtifactId=apex-app-archetype -DarchetypeVersion=3.4.0 \
+     -DarchetypeArtifactId=apex-app-archetype -DarchetypeVersion=RELEASE \
```

Comment:
While maven supports `RELEASE` for the version resolution, it may be better not to use it in favor of predictable version specification.